lms/static/js/verify_student/views/make_payment_step_view.js

```diff
@@ -105,10 +105,20 @@ var edx = edx || {};
                 self._getProductText( templateContext.courseModeSlug, templateContext.upgrade )
             );
 
-            // create a button for each payment processor
-            _.each(processors.reverse(), function(processorName) {
-                $( 'div.payment-buttons' ).append( self._getPaymentButtonHtml(processorName) );
-            });
+            if (processors.length === 0) {
+                // No payment processors are enabled at the moment, so show an error message
+                this.errorModel.set({
+                    errorTitle: gettext('No available payment processors'),
```

Comment:
Would it be OK to be less specific?

We’re currently experiencing technical problems.
Try the transaction again. If the problem persists, contact (email address or website).

If we can't direct users to an email address or website for help, I'd suggest this:

We’re currently experiencing technical problems.
Try the transaction again in a few minutes.

Also, does this text need to go in lines 212-213 of make_payment_step_view_spec.js as well?